fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1253,7 +1253,7 @@ dropFunctionStatement
     ;
 
 createFunctionStatement
-    : CREATE GLOBAL? functionType=(TABLE | AGGREGATE)? FUNCTION qualifiedName '(' typeList ')' RETURNS returnType=type (INTERMEDIATE intermediateType =  type)? properties?
+    : CREATE GLOBAL? functionType=(TABLE | AGGREGATE)? FUNCTION qualifiedName '(' typeList ')' RETURNS returnType=type properties? content=FUNCTION_CONTENT?
```

Comment:
Why are you removing `(INTERMEDIATE intermediateType = type)?`
Will it cause compatible issues?

Reply:
+1

Reply:
intermediateType won't be used any more